ShoppingList/Articles.cs

```diff
@@ -11,26 +12,24 @@ public class Articles
         public string NameOfThing;
         public int amount;
         public decimal price;
-        private decimal realPrice;
+        public List<Articles> article = new();
         public Articles() // Construct 
         {
             NameOfThing = "Not Define";
             amount = 0;
-            price = 0;
-            if(amount > 1)
-            {
-                realPrice = this.RealPrice();
-            }
-            else
-            {
-                realPrice = price;
-            }
+            price =  price * amount;
```

Comment:
price = price * amount does not do anything in this code as amount will always be 0 - so it is always 0 = 0 * 0